examples/zedo.amp.html

```diff
@@ -0,0 +1,41 @@
+<!doctype html>
```

Comment:
is there a special reason that we need a testing page for your ad? This file is not needed I think